packages/vuetify/src/components/VTextField/VTextField.ts

```diff
@@ -196,7 +196,7 @@ export default baseMixins.extend<options>().extend({
   },
 
   watch: {
-    labelValue: 'setLabelWidth',
+    // labelValue: 'setLabelWidth', // moved to mounted, see #11533
```

Comment:
Do you intent to leave this or just remaining dev code?

Reply:
Intended, can remove though if you like